src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -2002,6 +2009,50 @@ def setter(resource_group_name, vm_name, vm):
         identities = [MSI_LOCAL_ID]
 
     return _remove_identities(cmd, resource_group_name, vm_name, identities, get_vm, setter)
+
+
+# region VirtualMachines Identity
+def _remove_disk_encryption_set_identities(cmd, resource_group_name, name,
+                                           mi_system_assigned, mi_user_assigned, getter, setter):
+    IdentityType = cmd.get_models('DiskEncryptionSetIdentityType', resource_type=ResourceType.MGMT_COMPUTE)
+    remove_system_assigned_identity = mi_system_assigned is not None
+
+    resource = getter(cmd, resource_group_name, name)
+    if resource is None or resource.identity is None:
+        return None
```

Comment:
What is the function of this logic?

Reply:
> What is the function of this logic?

When the resource does not exist or the identity of the resource does not exist (identity type set to `None`), there is no need to perform remove operation. This logic is for this case.